Python/import.c

```diff
@@ -1021,6 +1021,13 @@ _imp_create_builtin(PyObject *module, PyObject *spec)
         return NULL;
     }
 
+    if (!PyUnicode_Check(name)) {
+        PyErr_Format(PyExc_TypeError,
+                     "name must be string, not %.200s",
+                     Py_TYPE(name)->tp_name);
+        return NULL;
```

Comment:
I'm not an expert on importlib, but I believe this needs a Py_DECREF(name), and it probably also needs a regression test.

Reply:
Done.